app/uk/gov/hmrc/fhregistrationfrontend/views/confirm_delete.scala.html

```diff
@@ -16,37 +16,36 @@
 
 @import org.joda.time.DateTime
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 
-@this(
-    mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
-)
+@this(layout: layout, viewHelpers: ViewHelpers)
 
 @(timeSaved: DateTime)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@title = @{
+@pageTitle = @{
   Messages("fh.delete_confirmation_page.header")
 }
 
-@mainTemplate(title = title) {
+@layout(pageTitle) {
 
-  <form action="@{uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.deleteUserData}" method="POST">
+    @viewHelpers.form(action = uk.gov.hmrc.fhregistrationfrontend.controllers.routes.Application.deleteUserData, 'novalidate -> "novalidate") {
     @helper.CSRF.formField
-    <h1>@title</h1>
+    <h1 class="govuk-heading-l">@pageTitle</h1>
 
-    <p>
-      <img src="./assets/images/File.png" align="left" style="margin: 0px 20px">
-    </p>
-    <p>
+    <p class="govuk-body">
       @Messages("fh.delete_confirmation_page.saved_on")
       @timeSaved.toLocalDate.toString("d MMMM yyyy")
       <br>
```

Comment:
We have to avoid using <br> tag, you can use the `<div class="govuk-!-padding-bottom-4"></div>` for creating a new line

Reply:
So remove the <br> and wrap it around the <div class="govuk-!-padding-bottom-4"></div> ?

Reply:
You can change the value "bottom-4" to any number desired , depending upon how much line space you need